### ISSUE TITLE
Continue training from saved/exported model?

### ISSUE THREAD

**Issue by jbmaxwell** [facebookresearch/audiocraft#344]
I've done some fine-tuning of `musicgen-small`, but I want to resume training from the checkpoint I exported (i.e., as `state_dict.bin`). I'm not sure how to do that. I tried: 

`continue_from: /home/james/src/somms/audiocraft/test_model/3genre` 

in my solver config, but that seems to be ignored. 
I also tried:

`continue_from: //pretrained/home/james/src/somms/audiocraft/test_model/3genre` 

since the `base.py` `StandardSolver` seems to have a `load_checkpoints` function... but those options just load from the last checkpoint in my `tmp` folder, with the previous (finished) scheduler, so although training begins the learning rate is zero.... 

If I use `--clear` in the dora call, then I lose my previous checkpoints... this should be easy (and presumably it is, but I can't find any documentation on it).

**Comment by GMPHL** [comment]
same problem

**Comment by escfrya** [comment]
try to specify full path to `state_dict.bin` 
in your case: `continue_from=/home/james/src/audiocraft/test_model/3genre/state_dict.bin`

**Comment by Frei-2** [comment]
@jbmaxwell @escfrya 
Hello, could you share some experiences on fine-tuning `musicgen-small` ? I have tried my best to follow the [MUSICGEN.md](https://github.com/facebookresearch/audiocraft/blob/main/docs/MUSICGEN.md) but when trying the finetuning command with the example dataset in `egs/example`, I encountered a lot of errors. The details of errors  can be seen [here](https://github.com/facebookresearch/audiocraft/issues/447#issuecomment-2582994865).

The command I used is as follows and I executed it under the root directory of this project, i.e., `audiocraft-main`.

```bash
dora run solver=musicgen/musicgen_base_32khz model/lm/model_scale=small continue_from=../cache_dir/musicgen-small conditioner=text2music dset=egs/example
```

Thanks in advance for any help !
